library/src/main/java/com/pengrad/telegrambot/model/request/ReplyKeyboardMarkup.java

```diff
@@ -9,31 +12,64 @@
 public class ReplyKeyboardMarkup extends Keyboard implements Serializable {
     private final static long serialVersionUID = 0L;
 
-    private final KeyboardButton[][] keyboard;
+    private final List<List<KeyboardButton>> keyboard;
     private boolean resize_keyboard;
     private boolean one_time_keyboard;
     private boolean selective;
 
-    public ReplyKeyboardMarkup(String[]... keyboard) {
-        this(keyboard, false, false, false);
-    }
-
-    public ReplyKeyboardMarkup(String[][] keyboard, boolean resize_keyboard, boolean one_time_keyboard, boolean selective) {
-        KeyboardButton[][] keyboardButtons = new KeyboardButton[keyboard.length][];
-        for (int i = 0; i < keyboard.length; i++) {
-            keyboardButtons[i] = new KeyboardButton[keyboard[i].length];
-            for (int j = 0; j < keyboard[i].length; j++) {
-                keyboardButtons[i][j] = new KeyboardButton(keyboard[i][j]);
-            }
-        }
-        this.keyboard = keyboardButtons;
+    public ReplyKeyboardMarkup(boolean resize_keyboard, boolean one_time_keyboard, boolean selective) {
+        this.keyboard = new ArrayList<>();
         this.resize_keyboard = resize_keyboard;
         this.one_time_keyboard = one_time_keyboard;
         this.selective = selective;
     }
 
+    public ReplyKeyboardMarkup() {
```

Comment:
This constructor is actually necessary, since the compiler can't choose between the other 2. But I remove the rest of the other constuctor / create methods (also in InlineKeyboardMarkup)